cloud_blobstore/__init__.py

```diff
@@ -1,8 +1,16 @@
+from datetime import datetime
+from enum import Enum
 import typing
-import types
 
 
-class PagedIter(typing.Iterable[str]):
+class BlobMetadataField(Enum):
+    CRC32C = "crc32c"
```

Comment:
Pls remove the CRC32C and ETAG fields.

otherwise the application code will need to know which cloud provider is being queried.

Each cloud_blobstore implementation should write to CHECKSUM.